### DISCUSSION TITLE
Apply-PnPTenantTemplate equivalent?

### DISCUSSION THREAD

**Discussion by sparkitect** [pnp/powershell#420] [Q&A] [answered]
How do we do the same thing as Apply-PnPTenantTemplate when we are using PnP.PowerShell vs. SharePointPnPPowershellOnline?

**Comment by gautamdsheth** [comment, marked as answer]
It is Invoke-PnPTenantTemplate.

https://pnp.github.io/powershell/cmdlets/Invoke-PnPTenantTemplate.html
